node_modules/oae-meetups/config/meeting.js

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
```

Comment:
Wow, this means we have been working on this integration for 3 years now, guys let's finish it :)